test/cbmc/proofs/MQTTAgent_CommandLoop/Makefile

```diff
@@ -48,6 +48,6 @@ PROOF_SOURCES += $(SRCDIR)/test/cbmc/stubs/agent_message_stubs.c
 PROOF_SOURCES += $(SRCDIR)/test/cbmc/stubs/agent_command_functions_stub.c
 PROOF_SOURCES += $(SRCDIR)/test/cbmc/stubs/core_mqtt_stubs.c
 
-PROJECT_SOURCES += $(SRCDIR)/source/mqtt_agent.c
```

Comment:
Lines 37-39 needs to be updated from __CPROVER_file_local_mqtt_agent_c to __CPROVER_file_local_core_mqtt_agent_c

Reply:
Thanks, updated